jetty-start/src/main/java/org/eclipse/jetty/start/FileInitializer.java

```diff
@@ -215,7 +215,7 @@ public boolean copyDirectory(Path source, Path destination) throws IOException
                 }
                 else if (!Files.exists(to))
                 {
-                    StartLog.log("COPY ", "%s to %s", _basehome.toShortForm(from), _basehome.toShortForm(to));
+                    StartLog.info("copy %s to %s", _basehome.toShortForm(from), _basehome.toShortForm(to));
```

Comment:
You used `cp` in `BaseBuilder:205` so I would be consistent - I like better "copy" than "cp".